initial-data-load/data/Sample/GeoData.yml

```diff
@@ -1,9 +1,38 @@
 - name: "Slovakia - regions"
   referenceId: "slovakiaRegions"
-  geoJSONFilename: "slovakia_regions.geojson"
+  geoDataFilename: "slovakia_regions.geojson"
+  format: "geojson"
+  featureIds: [{property: "name", values: ["Bratislavský", "Trnavský", "Trenčiansky", "Nitriansky", "Žilinský", "Banskobystrický", "Prešovský", "Košický"]}, { property: "fullname", values: ["Bratislavský kraj", "Trnavský kraj", "Trenčiansky kraj", "Nitriansky kraj", "Žilinský kraj", "Banskobystrický kraj", "Prešovský kraj", "Košický kraj"]}] 
+  attributeIds: [] 
```

Comment:
Should I left this here attributeIds: [], when there are no attributeIds or completely remove it?

Reply:
leave it as []

Reply:
done